linodecli/overrides.py

```diff
@@ -76,6 +76,46 @@ def handle_image_replicate(operation, output_handler, json_data) -> bool:
     return image_replicate_output(json_data)
 
 
+@output_override("placement", "group-view", OutputMode.table)
```

Comment:
I didn't override the `placement groups-list` output here, because I think the list view is already huge itself if user have multiple PGs, and we can try to keep it simple. User can check the singe group view for members details in this case. Let me know if you have different thoughts!